contrib/net/http/trace.go

```diff
@@ -28,6 +28,9 @@ type ServeConfig struct {
 	Resource string
 	// QueryParams specifies any query parameters that be appended to the resulting "http.url" tag.
 	QueryParams bool
+	// RouteParams specifies framework-specific route parameters (e.g. for route /user/:id coming
+	// in as /user/123 we'll have {"id": "123"}). This field is optional and monitored by appsec.
```

Comment:
```suggestion
	// in as /user/123 we'll have {"id": "123"}). This field is optional and is used for monitoring
	// by AppSec.
```
It's also worth mentioning that it's used specifically for that. In the current form we are saying that it's "monitored by appsec" but it's not clear if it has other uses to.

Is there any point in mentioning more details, as in how AppSec monitors these? I'm thinking that maybe some people who may be using `TraceAndServe` independently for their own handlers, might be able to benefit from AppSec by using these fields with some other framework which is not supported by us. Is it also worth mentioning that AppSec must be enabled for these to be taken into consideration? For example: if someone passes `RouteParams` but has AppSec disabled, will it be taken into account?

Reply:
I did a simple addition here: https://github.com/DataDog/dd-trace-go/pull/1106/commits/7ff222dc151448766c0db35587188a05090466bd

No need to further explain how AppSec monitors them, as it depends on the security rules, that are exposed and detailed in our UI.